### PR TITLE
PREAPPS-4135: Fix zm-x-zimlet-template-default after preact x upgrade

### PR DESCRIPTION
**Pull request by yashrajpchavda**
- updated necessary code changes after graphql-preactx upgrade

### CODE REVIEW

File: template/package.json [hunk 1]
```diff
@@ -11,25 +11,28 @@
     "watch": "zimlet watch",
     "start": "zimlet watch",
     "package": "zimlet package",
-    "test": "eslint src",
+    "lint": "eslint src",
+    "lint:fix": "eslint src --fix",
```
Comment (silentsakky) [inline template/package.json]: `npm run lint -- --fix`

File: template/package.json [hunk 2]
```diff
@@ -11,25 +11,28 @@
     "watch": "zimlet watch",
     "start": "zimlet watch",
     "package": "zimlet package",
-    "test": "eslint src",
+    "lint": "eslint src",
```
Comment (silentsakky) [inline template/package.json:14]: we need to update this in circleci config

File: template/package.json [hunk 3]
```diff
@@ -11,25 +11,28 @@
     "watch": "zimlet watch",
     "start": "zimlet watch",
     "package": "zimlet package",
```
Comment (silentsakky) [inline template/package.json]: add required properties here

File: template/package.json (hunk 2 above)
Comment (lavegupta) [reply template/package.json:14]: https://github.com/Zimbra/zm-x-zimlet-sideloader/pull/2/files#diff-1d37e48f9ceff6d8030570cd36286a61R48

File: template/package.json (hunk 3 above)
Comment (lavegupta) [reply template/package.json]: https://github.com/Zimbra/zm-x-zimlet-sideloader/pull/2/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R13

**Review by silentsakky — APPROVED** [review]

**Pull request merged**